data/trainers/parties.asm

```diff
@@ -3,10 +3,10 @@ INCLUDE "constants.asm"
 ; Trainer data structure:
 ; - db "NAME@", TRAINERTYPE_* constant
 ; - 1 to 6 Pokémon:
-;    * for TRAINERTYPE_NORMAL:     db level, species
-;    * for TRAINERTYPE_ITEM:       db level, species, item
-;    * for TRAINERTYPE_MOVES:      db level, species, 4 moves
-;    * for TRAINERTYPE_ITEM_MOVES: db level, species, item, 4 moves
+;	* for TRAINERTYPE_NORMAL:	 db level, species
+;	* for TRAINERTYPE_ITEM:	   db level, species, item
+;	* for TRAINERTYPE_MOVES:	  db level, species, 4 moves
+;	* for TRAINERTYPE_ITEM_MOVES: db level, species, item, 4 moves
```

Comment:
This is part of the reason we do spaces for alignment

Reply:
Oh, okay. I was trained to do tabs for asm.